### PR TITLE
Add usage documentation to Python tools distribution

### PR DESCRIPTION
**Pull request by soltanmm**
What it says on the tin.

### CODE REVIEW

File: tools/distrib/python/grpcio_tools/README.rst [hunk 1]
```diff
@@ -126,3 +126,13 @@ Help, I ...
   GCC 6.0), this is probably a bug where GCC chokes on constant expressions
   when the :code:`-fwrapv` flag is specified. You should consider setting your
   environment with :code:`CFLAGS=-fno-wrapv` or using clang (:code:`CC=clang`).
+
+Using
```
Comment (kpayson64) [inline tools/distrib/python/grpcio_tools/README.rst]: Nit: Using->Usage

**Comment by kpayson64** [comment]
I just wanted to verify that you don't need to specify a .proto file or directory file to compile, I usually invoke protoc roughly as 
```protoc -I <Include> -python-out=<out> -python_grpc_out=<out> <Some proto file>```

**Comment by soltanmm** [comment]
Nits (and brain snafu) addressed.

**Comment by kpayson64** [comment]
lgtm

**Comment by nathanielmanistaatgoogle** [comment]
Luhguhtum; marking for merge.

**Pull request merged**